packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -104,7 +169,29 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
   S find<S>(Offset regionOffset);
 
   /// Override this method to upload this layer to the engine.
-  void addToScene(ui.SceneBuilder builder, [Offset layerOffset = Offset.zero]);
```

Comment:
DBC

This is a breaking change, and will require a manual roll into Fuchsia. See: https://logs.chromium.org/logs/fuchsia/buildbucket/cr-buildbucket.appspot.com/8931658542863723648/+/steps/build/0/steps/build_fuchsia/0/steps/ninja/0/stdout

Reply:
A patch was actually ready to fix it since 10/11: https://fuchsia-review.googlesource.com/c/topaz/+/211549

However, I forgot to add tag back then and now the presubmit somehow always gets red. Do you or @rmacnak-google know what's wrong with topaz-x64-perf-dawson_canyon there?

Reply:
Update: https://fuchsia-review.googlesource.com/c/topaz/+/211549 has landed so it must be a flaky test. Can we kick an auto roll into Fuchsia to see if succeeds this time?

Reply:
Did we make a a breaking change announcement for this?
This breaks anyone that extends Layer, e.g it broke #23412